plugin/entryAttributes.go

```diff
@@ -160,11 +160,15 @@ func (a *EntryAttributes) SetSize(size uint64) *EntryAttributes {
 	return a
 }
 
-// Meta returns the entry's metadata. All entries have a
-// meta attribute, which by default is an empty map.
+// Meta returns the portion of the entry's metadata that's returned by the
```

Comment:
Main thing I can't tell from looking at these changes is whether Meta will still be refreshed with new data when some cache expires. Also should Meta be cached at all?

Reply:
Since it's tied to the `List` endpoint, it will be refreshed when the entry's refreshed.

I don't think we need to cache meta. For entries where `meta` represents the full metadata, we could probably remove the `Metadata` method (e.g. EC2 instances).